draft-ietf-core-senml.md

```diff
@@ -518,14 +519,27 @@ by the application context.
 
 ### Time
 
-If either the Base Time or Time value is missing, the missing
-field is considered to have a value of zero. The Base Time and
-Time values are added together to get the time of measurement. A time
-of zero indicates that the sensor does not know the absolute time and
-the measurement was made roughly "now". A negative value is used to
-indicate seconds in the past from roughly "now". A positive value is
-used to indicate the number of seconds, excluding leap seconds, since
-the start of the year 1970 in UTC.
+If either the Base Time or Time value is missing, the missing field is
+considered to have a value of zero. The Base Time and Time values are
+added together to get the time of measurement. A time of zero
+indicates that the sensor does not know the absolute time and the
+measurement was made roughly "now". A negative value is used to
+indicate seconds in the past from roughly "now". 
+
+Positive values up to 2^28 indicate seconds in the future from "now".
+These can be used e.g., for actuation use when the desired change
+should happen in the future but the sender or the receiver does not
+have accurate time available.
+
+Values greater than 2^28 represent an absolute time relative to the
+Unix epoch (1970-01-01T00:00Z in UTC time) and the time is counted
+same way as the Portable Operating System Interface (POSIX) "seconds
+since the epoch" {{TIME_T}}.
+
+Because time values up to 2^28 are used for presenting time relative
+to "now" and Time and Base Time are added together, care must be taken
+to ensure that the sum does not inadvertently become greater and 2^28
+(i.e., absolute time) when relative time was intended to be used.
```

Comment:
This sentence is weird.

Reply:
Any suggestions? The idea is to have the same disclaimer as before.

Reply:
maybe s/become greater and/become greater than/?
(Actually, I'd prefer "reach or exceed".)